R/applications.R

```diff
@@ -316,6 +322,52 @@ showLogs <- function(
   }
 }
 
+#' @rdname showLogs
+#' @export
+getLogs <- function(
+  appPath = getwd(),
+  appFile = NULL,
+  appName = NULL,
+  account = NULL,
+  server = NULL,
+  entries = 50
+) {
+  # determine the log target and target account info
+  deployment <- findDeployment(
+    appPath = appPath,
+    appName = appName,
+    server = server,
+    account = account
+  )
+  accountDetails <- accountInfo(deployment$account, deployment$server)
+  client <- clientForAccount(accountDetails)
+  application <- getAppByName(client, accountDetails, deployment$name)
+
+  payload <- client$getLogs(application$id, entries, format = "json")
+  payload
+
+  # entries in logs$results have unstable field order.
+  ordered <- lapply(payload$results, function(row) row[order(names(row))])
+  combined <- do.call(rbind, ordered)
+
+  # JSON loves wrapped values.
+  unwrapped <- lapply(colnames(combined), function(nm) unlist(combined[, nm]))
+  df <- as.data.frame(unwrapped)
+  names(df) <- colnames(combined)
```

Comment:
```suggestion
  ordered <- lapply(payload$results, function(row) as.data.frame(row[order(names(row))]))
  df <- do.call(rbind, ordered)
```

I think if you convert to data frame earlier then some of the subsequent parsing becomes unnecessary